tests/centos10-build/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM quay.io/centos/centos:stream10-development
```

Comment:
Can we call this a Containerfile?

Reply:
we are using it with Docker: https://github.com/containers/crun/pull/1465/files#diff-245392b692a50c38ecab4381b118862db514035c10983f3bd4f4b7f1f4be4692R155-R157